.github/README.md

```diff
@@ -107,73 +115,97 @@ $thePayClient = new \ThePay\ApiClient\TheClient(
 
-The payment method can be preselected on your side and simply added as payment parameter to the API.
-Otherwise, the customer will be presented with payment method selection on visiting ThePay gate through generated link.
+#### Payment method
+You have two options, regarding whether the payment method is preselected or not:
+- Payment method preselected in your e-shop (upon payment creation or later)
+- Payment method NOT preselected - the customer will select payment method at ThePay gateway
 
-The payment link is returned to you in a response, upon calling the API endpoint for payment creation.
+The payment method can be preselected on your side simply by adding a parameter to the API call for payment creation.
+By using this approach you can fully customize how you want to display the payment methods in your e-shop.
+In addition, it is also possible to change the payment method from your side after it has been already created, if you need to do so for some reason.
 
-#### Payment amount is unchangeable
-In case your order amount changes, a new payment needs to be created.
+If you do not preselect the payment method, the customer will be presented with payment method selection upon
+visiting ThePay gateway through the generated link.
 
-#### Payment flow and changes
-You should always create only one payment (with its unique UID) for each order in your e-shop. You should never create new payments, except when changing the payment amount.
+Note that even if the payment method is set from your side, the customer can still change it in ThePay gateway after redirection — unless this is explicitly forbidden.
+To prevent changes, you must specify the setting when creating the payment by adding the appropriate parameter to the payment creation API call.
 
-#### TL;DR - summary
-There is only one supported way to create payment:
+You can find more examples of the mentioned use cases here:
+- [Preselection before payment creation](../doc/create-payment.md)
+- [Payment method change after payment was created](../doc/change-payment-method-of-payment.md)
+- [Disable change of payment method](../doc/payment-disable-payment-method-change.md)
 
-- API - creating the payment through API call (selection of payment method either in e-shop or ThePay gate)
+#### Payment amount is unchangeable
 
-Always create only one payment for your order for all payment creation options, unless you need to change the payment amount. In that case, consider it a whole new payment.
+Please note that the amount for which the payment was created cannot be changed later.
 
-For more examples see [create-payment.md](../doc/create-payment.md)
+This means if the order is updated on your side and the final amount of the payment changes,
+you will need to initiate a new payment via a new API call with the updated amount (and a new unique identifier).
 
-[See how to make TheClient](#theclient-instance)
+Once a payment is created, it is not possible to modify the amount.
```

Comment:
This sentence has the same meaning as the one above: https://github.com/ThePay/api-client/pull/90/files#diff-438478e7713ffdb96be895cd2cba03144f30c04f1442ae246acdeb2f1c13efd5R167
```markdown
Please note that the amount for which the payment was created cannot be changed later.
```